pom.xml

```diff
@@ -124,6 +124,57 @@
         </plugins>
       </build>
     </profile>
+    <profile>
```

Comment:
do we need this profile? this should only be needed for the jenkins parent pom not plugin parent afaik

Reply:
I don't think we need the profile?

The plugin pom and HPI should do the correct thing?

Reply:
May be unneeded, but based on my experience, I had to add it when updating the maven stapler plugin to not produce Java 11+ binaries only. Let's see 🤞🏻

Reply:
I am fine with breaking java 8 compatibility, would mean we could do a lot of code cleanup! 🗡️

Reply:
Would be nice to have JRE metric for plugins:
Much like [Plugin Versions by Jenkins Version](https://stats.jenkins.io/pluginversions)

Reply:
stapler plugin would need it but an actual plugin does not need this

Reply:
> stapler plugin would need it but an actual plugin does not need this



Removed it, only change that's left is the Jenkinsfile and GH action modification.